parson.c

```diff
@@ -799,7 +800,8 @@ static _Ptr<JSON_Value> parse_value(_Ptr<_Nt_array_ptr<const char>> string, size
 }
 
 static _Ptr<JSON_Value> parse_object_value(_Ptr<_Nt_array_ptr<const char>> string, size_t nesting) {
-    _Ptr<JSON_Value> output_value = NULL, new_value = NULL;
+    _Ptr<JSON_Value> output_value = NULL;
```

Comment:
Moving declarations of `_Ptr` variables in this file to separate lines does not result in any fewer compiler errors or warnings. These can be declared on the same line.